src/CheckBoxFilter.webmodeler.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, createElement } from "react";
+import * as classNames from "classnames";
+
+import { Alert } from "./components/Alert";
+import { CheckboxFilter } from "./components/CheckBoxFilter";
+import { ContainerProps } from "./components/CheckBoxFilterContainer";
+import { Utils, parseStyle } from "./utils/ContainerUtils";
+
+// tslint:disable-next-line class-name
+export class preview extends Component<ContainerProps, {}> {
+    constructor(props: ContainerProps) {
+        super(props);
+    }
+
+    render() {
+        return createElement("div",
+            {
+                className: classNames("widget-checkbox-filter", this.props.class),
+                style: parseStyle(this.props.style)
+            },
+            this.renderAlert(),
+            createElement(CheckboxFilter, {
+                handleChange:  () => { return; },
+                isChecked: this.props.defaultChecked
+            })
+        );
+    }
+
+    private renderAlert() {
+        const errorMessage = Utils.validateProps({
```

Comment:
I think if you inline this, it will still look great

Reply:
Thanks. Done